bugwarrior/docs/using.rst

```diff
@@ -15,6 +18,41 @@ adding the following to your crontab::
     DISPLAY=:0
     */15 * * * *  /usr/bin/bugwarrior-pull
 
+
+systemd timer
+-------------
+
+If you would prefer to use a systemd timer to run ``bugwarrior-pull`` on a
+schedule, you can create the following two files::
+
+    $ cat ~/.config/systemd/user/bugwarrior-pull.service 
+    [Unit]
+    Description=bugwarrior-pull
+
+    [Service]
+    Environment="DISPLAY=:0"
+    ExecStart=/usr/bin/bugwarrior-pull
+    Type=oneshot
+
+    [Install]
+    WantedBy=default.target
+    $ cat ~/.config/systemd/user/bugwarrior-pull.timer 
+    [Unit]
+    Description=Run bugwarrior-pull weekly and on boot
```

Comment:
Good call - on my own machine (where I copied this from) I run it weekly, but I wanted it to be hourly to match the cron example. I'll fix that right up!